include/console_bridge/console.h

```diff
@@ -34,12 +34,12 @@
 
 /* Author: Ryan Luna, Ioan Sucan */
 
-#ifndef CONSOLE_BRIDGE_CONSOLE_
-#define CONSOLE_BRIDGE_CONSOLE_
+#ifndef INCLUDE_CONSOLE_BRIDGE_CONSOLE_H_
+#define INCLUDE_CONSOLE_BRIDGE_CONSOLE_H_
 
 #include <string>
 
-#include "console_bridge_export.h"
+#include "./console_bridge_export.h"
```

Comment:
Why is this necessary?

Reply:
cpplinter shows the following error:

> Include the directory when naming .h files  [build/include] [4]

another workaround could be this one, but then this header will be consider as a C system header:

```
#include <console_bridge_export.h>
```

Reply:
Ack, thanks for the reply. For reference it seems that that cpplint check is some check relative to Google include guidelines that I do not think really apply to general CMake projects (see https://github.com/ament/ament_lint/blob/master/ament_cpplint/ament_cpplint/cpplint.py#L4485), but if the workaround is working and I don't think it is a big problem.